docs/components.md

```diff
@@ -1,28 +1,25 @@
 ---
 nav_order: 2
-title: Components
+title: Hardware Setup
 ---
 
-# Components
-
-Here we have detailed the components we used for this project, and why we selected them
-
-### Raspberry Pi X Model X
-
-TBC
-
-### Camera Module
-
-TBC
-
-## Overall Cost
-
-Cost is a critical consideration in developing a new product such as this, and played a key role in our choice of components.
-
-The overall cost breakdown of the system is as follows:
-
-Component | Cost
-----------|-----
-Raspberry Pi X... | £X
-... | ...
-Total | £X
+## You will need:
+- Raspberry Pi
+  - We use a Raspberry Pi 4B for testing, but other versions may work.
+- Raspberry Pi Camera Module
+- Screen/monitor
+- SD Card
+  - Minimum recommended storage size is 8GB
+  - Note that Raspberry Pi model 3 or higher takes a micro SD card
+- USB type C power supply
+- Optionally, a case for the Raspberry Pi and camera
+
+We estimate that this will cost around £150 for all components.
+
+## Assembly
+This section covers the hardware assembly. For software setup checkout the [software setup guide (TBC)]().
+
+1. Attach the camera module to the CSI port. This is the slot closest to the Ethernet (or furthest from the SD card). Lift up the plastic toggle and slide in the camera ribbon cable. Ensure the contacts on the cable face in the correct direction. Push down the toggle softly to fix the ribbon cable in place.
+1. Attach the screen or monitor. You will either attach this via a HDMI port (on the side) or the other ribbon cable port.
+1. Insert the flashed SD card (see [here(TBC)]() for insturctions on flashing).
```

Comment:
Minor things - these are all numbered step "1" and "insturctions" -> "instructions"

Reply:
Typo fixed. I think in Markdown you're supposed to number by starting all lines with `1.` and then it automatically puts in the correct numbering. That stops any errors were lines are moved or removed causing the numbering to in the wrong order.

Reply:
Ooh okay didn't actually know that thanks!